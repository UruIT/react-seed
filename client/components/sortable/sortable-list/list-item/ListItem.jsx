import React from 'react';
import { SortableElement } from 'react-sortable-hoc';
const ListItem = SortableElement(({ children }) => (
	<div>{children}</div>
));

export default ListItem;
