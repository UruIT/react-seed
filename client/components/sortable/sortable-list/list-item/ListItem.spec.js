import React from 'react';
import renderer from 'react-test-renderer';

import ListItem from './ListItem';

describe('<ListItem />', () => {
	it('snapshot', () => {
		const tree = renderer
			.create(
				<ListItem>
					<p>TEST</p>
				</ListItem>
			)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
