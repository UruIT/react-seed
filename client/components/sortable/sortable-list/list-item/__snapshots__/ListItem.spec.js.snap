// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListItem /> snapshot 1`] = `
<div>
  <p>
    TEST
  </p>
</div>
`;
